### PR TITLE
Zeus - Add ability to unload cargo

### PR DESCRIPTION
**Pull request by mharis001**
**When merged this pull request will:**
- Add ability to unload cargo from a vehicles zeus attribute display
cargo_unload

### CODE REVIEW

File: addons/zeus/functions/fnc_ui_attributeCargo.sqf [hunk 1]
```diff
@@ -19,18 +19,75 @@
...
+// Init unload button
+private _button = _control controlsGroupCtrl 80087;
+
+private _fnc_onButtonUnload = {
+    params ["_button"];
+
+    // Validate vehicle
+    private _vehicle = GETMVAR(BIS_fnc_initCuratorAttributes_target,objNull);
+    if (isNull _vehicle || {!alive _vehicle}) exitWith {
+        LOG("Vehicle deleted or killed, cannot unload");
```
Comment (SilentSpike) [inline addons/zeus/functions/fnc_ui_attributeCargo.sqf]: It's a been a while, but is log a valid macro? I ask because it's an sqf command 🤔

Comment (jonpas) [reply addons/zeus/functions/fnc_ui_attributeCargo.sqf]: Yes it is.

Comment (dedmen) [reply addons/zeus/functions/fnc_ui_attributeCargo.sqf]: @SilentSpike Don't remember
`#define true false` fun? ^^
The preprocessor doesn't know what SQF is. It just does it's job and hands the result off to others.

Comment (SilentSpike) [reply addons/zeus/functions/fnc_ui_attributeCargo.sqf]: 😆 Yeah I remembered, that's why I wasn't sure if we'd actually hijacked that command

**Pull request merged**